File: Farthest Distance.py
def furthest_distance(points):
    result  = 0
    for i in range(len(points)-1):
        for j in range(i+1,len(points)):
            value = ((points[i][0]-points[j][0])**2+(points[i][1]-points[j][1])**2)**0.5
            if value > result:
                result = value
    return round(result,2)

File: Farthest Distance/Farthest Distance.js
function furthestDistance(arr){
   let min_v = -Infinity
   for(let i=0;i<arr.length-1;i++){
     for(let j=i+1;j<arr.length;j++){
       let tmp = Math.round((Math.sqrt((arr[i][0]-arr[j][0])**2+(arr[i][1]-arr[j][1])**2))*100)/100
       if(tmp > min_v )min_v = tmp
     }
   }
  return min_v
}
